setup.py

```diff
@@ -1,12 +1,35 @@
 #!/usr/bin/env python
 
-from distutils.core import setup
-
-setup(name='BeatCop',
-      version='1.0',
-      description='Cluster singleton manager',
-      long_description=open('README.md').read(),
-      url='https://github.com/Luluvise/BeatCop',
-      requires=['redis(>=2.9.1)'],
-      scripts=['beatcop.py'],
+from __future__ import print_function
+
+from setuptools import setup
+
+
+long_description = '...'
+for _fln in ('README.rst', 'README.md'):
+    try:
+        with open(_fln) as fo:
+            long_description = fo.read()
+    except Exception as exc:
+        pass
+    else:
+        break
+
+
+setup_kwargs = dict(
+    name='BeatCop',
+    version='1.0',
+    description='Cluster singleton manager',
+    long_description=open('README.md').read(),
```

Comment:
Here it's simply read from the file again. Or is this some setup.py black magic?